apps/src/templates/instructions/codeReviewV2/CodeReviewCommentEditor.jsx

```diff
@@ -64,21 +97,91 @@ const CodeReviewCommentEditor = ({addCodeReviewComment}) => {
   );
 };
 
+const toggleBlock = (editor, format) => {
+  const isActive = isBlockActive(editor, format);
+
+  if (isActive) {
+    Transforms.unwrapNodes(editor, {
```

Comment:
this can be done as a follow up, but we should add some comments explaining what this logic is doing